client/app/nonComp/components/TaskTableColumns.jsx

```diff
@@ -10,11 +10,11 @@ export const claimantColumn = () => {
   };
 };
 
-export const veteranParticipantIdColumn = () => {
+export const veteranSsnColumn = () => {
   return {
-    header: 'Veteran Participant Id',
-    valueFunction: (task) => task.veteranParticipantId,
-    getSortValue: (task) => task.veteranParticipantId
+    header: 'Veteran SSN',
+    valueFunction: (task) => task.veteranSSN,
+    getSortValue: (task) => task.veteranSSN
   };
 };
 
```

Comment:
```suggestion
export const veteranParticipantIdColumn = () => {
  return {
    header: 'Veteran Participant Id',
    valueFunction: (task) => task.veteranParticipantId,
    getSortValue: (task) => task.veteranParticipantId
  };
};
```

Would you mind adding back in this column so that we can toggle between the two once the FeatureToggle is in place?

Reply:
This has been added back.